src/parser/core/Parser.js

```diff
@@ -434,10 +441,13 @@ class Parser {
 	 */
 	scrollTo(handle) {
 		const module = this.modules[handle]
-		scroller.scrollTo(module.constructor.title, {
-			offset: -50,
-			smooth: true,
-		})
+		const element = document.getElementById(module.constructor.title)
```

Comment:
Changing to an `id` also had this nice side effect, instead of having to use a `querySelector` or somehow put the Provider at an even higher level and having to keep track of names in the Provider.